modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -45,31 +45,19 @@ package object engine {
     */
   def switch(q: EventQueue)(st: Status): Engine[QState] =
     modify(QState.status.set(_, st)) *>
-    whenM (st == Status.Running) (prime *> execute(q)) *>
+    whenM (st == Status.Running) (execute(q)) *>
```

Comment:
[`whenM`](https://oss.sonatype.org/service/local/repositories/releases/archive/org/scalaz/scalaz_2.11/7.2.5/scalaz_2.11-7.2.5-javadoc.jar/!/index.html#scalaz.Applicative@whenM[A](cond:Boolean)(f:=>F[A]):F[Unit]) is a `Scalaz` function.

I could use explicit imports for `scalaz`, as a code reader I acknowledge is way better than wildcard imports. I think it wouldn't take me so much time now that I'm more familiar with the library. Do you think it's worth it?